docs/escape_support.md

```diff
@@ -0,0 +1,112 @@
+|            |             |   `2004`                                          |
+| `CSI I`    | IMPLEMENTED |                                                   |
+| `CSI J`    | IMPLEMENTED |                                                   |
+| `CSI K`    | IMPLEMENTED |                                                   |
+| `CSI L`    | IMPLEMENTED |                                                   |
+| `CSI l`    | PARTIAL     | See `CSI h` for supported modes                   |
+| `CSI ? l`  | PARTIAL     | See `CSI ? h` for supported modes                 |
+| `CSI M`    | IMPLEMENTED |                                                   |
+| `CSI m`    | PARTIAL     | Colon separators are not supported                |
+| `CSI n`    | IMPLEMENTED |                                                   |
+| `CSI P`    | IMPLEMENTED |                                                   |
+| `CSI SP q` | PARTIAL     | No blinking support                               |
+| `CSI r`    | IMPLEMENTED |                                                   |
+| `CSI S`    | IMPLEMENTED |                                                   |
+| `CSI s`    | IMPLEMENTED |                                                   |
+| `CSI T`    | IMPLEMENTED |                                                   |
+| `CSI t`    | PARTIAL     | Only parameters `22` and `23` are supported       |
+| `CSI u`    | IMPLEMENTED |                                                   |
+| `CSI X`    | IMPLEMENTED |                                                   |
+| `CSI Z`    | IMPLEMENTED |                                                   |
+
+### OSC (Operating System Command) - `ESC ]`
+
+| ESCAPE    | STATUS      | NOTE                                               |
+| --------- | ----------- | -------------------------------------------------- |
+| `OSC 0`   | IMPLEMENTED |                                                    |
+| `OSC 1`   | REJECTED    | Tabs are not supported                             |
+| `OSC 2`   | IMPLEMENTED |                                                    |
+| `OSC 4`   | IMPLEMENTED |                                                    |
+| `OSC 10`  | IMPLEMENTED |                                                    |
+| `OSC 11`  | IMPLEMENTED |                                                    |
+| `OSC 12`  | IMPLEMENTED |                                                    |
+| `OSC 50`  | IMPLEMENTED | Only `CursorShape` is supported                    |
+| `OSC 52`  | IMPLEMENTED | Only Clipboard and primary selection supported     |
+| `OSC 104` | IMPLEMENTED | Only Clipboard and primary selection supported     |
+| `OSC 110` | IMPLEMENTED |                                                    |
+| `OSC 111` | IMPLEMENTED |                                                    |
+| `OSC 112` | IMPLEMENTED |                                                    |
```

Comment:
In general, I guess it's nice to point to a things where those OSC came from and some references where to look them up. Maybe some links to xterm docs, etc?

Reply:
I'm not sure if we want to link directly to just XTerm docs, since in many cases they're also just an aggregation of other sources? In our wiki we have a list with a bunch of different resources, that's probably what people should be looking at.

But I'm not sure if we want to link to any of that? Since it's not really the purpose of this document to explain what these escapes are at all, just which ones are supported.